frontend/general/categories/css.md

```diff
@@ -1,3 +1,5 @@
 # CSS
 
-* Box-Model
\ No newline at end of file
+* Box-Model
+* BEM
+* Flex-Box
```

Comment:
If we do not intend to include Sass questions in a different section, we can also add some questions about Sass and postCSS.